drivers/bmp180/bmp180_saul.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2016 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_bmp180
+ * @{
+ *
+ * @file
+ * @brief       SAUL adaption for BMP180 device
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <string.h>
+
+#include "saul.h"
+#include "bmp180.h"
+#include "xtimer.h"
+
+static int32_t temperature, pressure;
+
+static int check_and_read_temperature(void *dev, phydat_t *res)
+{
+    bmp180_t *d = (bmp180_t *)dev;
+
+    bmp180_read_temperature(d, &temperature);
+    res->val[0] = temperature;
+    res->unit = UNIT_TEMP_C;
+    res->scale = -1;
+    return 1;
+}
+
+static int check_and_read_pressure(void *dev, phydat_t *res)
+{
+    bmp180_t *d = (bmp180_t *)dev;
+
+    bmp180_read_pressure(d, &pressure);
+
+    res->val[0] = pressure/10;
+    res->unit = UNIT_PA;
+    res->scale = 1;
```

Comment:
Why did you set the scale to `1`? I'm wondering because the [doc](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/phydat.h#L136) says this results in a factor `10^scale`shouldn’t it be `-1` because you divide by 10?